### PR TITLE
Release changes for 8.1 (part 2)

### PR DESCRIPTION
**Pull request by chkim-usgs**
## Description

Adds code.json and pins gsl to 2.6

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Infrastructure change (changes to things like CI or the build system that do not impact users)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Review by acpaquette — APPROVED** [review]

File: code.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "name": "ISIS3",
     "organization": "U.S. Geological Survey",
     "description": "GitHub code repository for the Integrated Software for Imagers and Spectrometers (ISIS)",
-    "version": "8.0.0",
+    "version": "8.1.0_RC1",
```
Comment (acpaquette) [inline code.json]: This should be it's own code entry

**Review by acpaquette — CHANGES_REQUESTED** [review]

File: code.json [hunk 2]
```diff
@@ -84,6 +84,50 @@
...
+    "description": "GitHub code repository for the Integrated Software for Imagers and Spectrometers (ISIS)",
+    "version": "8.1.0_RC1",
+    "status": "Production",
+
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://code.usgs.gov/astrogeology/isis/-/raw/7.1.0/LICENSE.md"
```
Comment (acpaquette) [inline code.json]: This should be changed to the right version

File: code.json [hunk 3]
```diff
@@ -84,6 +84,50 @@
...
+      "Data Processing"
+    ],
+
+    "languages": [
+      "C++"
+    ],
+
+    "contact": {
+      "name": "Dr. Jason Laura",
+      "email": "jlaura@usgs.gov"
```
Comment (acpaquette) [inline code.json]: This should probably be someone on the dev team. I think per release we just make it the person who made the release

File: code.json (hunk 2 above)
Comment (chkim-usgs) [reply code.json]: Looks like the above code entry URLs haven't been updated either, I can change that

Comment (acpaquette) [reply code.json]: Sure, I think that would be a good idea

**Review by acpaquette — APPROVED** [review]

**Pull request merged**